support/pygments/koka.py

```diff
@@ -25,19 +25,19 @@ class KokaLexer(RegexLexer):
 
     keywords = [
         'infix', 'infixr', 'infixl',
-        'module', 'import', 'as',
-        'pub', 'abstract',
+        'module', 'import', 'as', 'mod'
+        'pub', 'abstract', 'ctx'
         'type', 'struct', 'alias', 'effect', 'con',
         'forall', 'exists', 'some',
         'fun', 'fn', 'val', 'var', 'extern',
         'if', 'then', 'else', 'elif', 
         'match', 'return', 'with', 'in',
         'handle', 'handler', 'mask',
         'ctl', 'final', 'raw',
-        'override', 'named',
+        'override', 'named', 'scoped',
         'interface', 'break', 'continue', 'unsafe',
         'co', 'rec', 'open', 'extend', 'behind',
-        'linear', 'value', 'reference', 
+        'linear', 'value', 'reference',
```

Comment:
Not entirely sure about this, but isn't `ref` (a few lines down from here) a keyword too rather than a `builtin`?

Reply:
Actually it is a built-in identifier. It is also the way to create a mutable reference cell.

Reply:
Oh right, so it has a different meaning depending on the context. I guess that is not easy to encode in the pygments file, so the identifier was chosen here.

Reply:
Yeah, it's a tricky one.